demos/appyx-navigation/src/main/kotlin/com/bumble/appyx/navigation/node/backstack/BackStackNode.kt

```diff
@@ -29,12 +35,6 @@ import com.bumble.appyx.navigation.node.ParentNode
 import com.bumble.appyx.navigation.node.node
 import com.bumble.appyx.navigation.ui.TextButton
 import com.bumble.appyx.navigation.ui.appyx_dark
-import com.bumble.appyx.transitionmodel.backstack.BackStack
-import com.bumble.appyx.transitionmodel.backstack.BackStackModel
-import com.bumble.appyx.transitionmodel.backstack.operation.newRoot
-import com.bumble.appyx.transitionmodel.backstack.operation.pop
-import com.bumble.appyx.transitionmodel.backstack.operation.push
-import com.bumble.appyx.transitionmodel.backstack.operation.replace
 import kotlinx.parcelize.Parcelize
 import kotlin.random.Random
 
```

Comment:
`BackStackNode` refers to `NavTarget` but shouldn't we rename it to `InteractionTarget`?